Frameworks/UIKit.Xaml/ScrollView.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ScrollView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <!-- Layer content which is a image is behind any sublayers-->
+    <Image x:Name="LayerContentImage"></Image>
```

Comment:
This is still more expensive than just adding the Image the first time it's requrested (like we do in Button and Label).

---
In reply to: [106707117](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106707117) [](ancestors = 106707117)